docs/articles/new-expensify/expenses-&-payments/Pay-an-invoice.md

```diff
@@ -56,4 +55,4 @@ You can add additional payment methods to your [Wallet](https://help.expensify.c
 You will be prompted to choose a payment method when paying future invoices. 
 {% include faq-end.md %}
 
-</div>
+{% include faq-end.md %}
```

Comment:
Can you revert this? We already have a `{% include faq-end.md %}` on line 56 here, and we also need the `</div>` tag in order to close the opening `<div>` tag from line 5 of this file.

Reply:
Ah okay, I updated the article! TY!